### PR TITLE
Release v1.3

### PR DESCRIPTION
**Pull request by xNombre**

### CODE REVIEW

**Comment by xNombre** [comment]
Ive added a rule about `ro.control_privapp_permissions` please confirm if your phone boots fine without it.

File: README.md [hunk 1]
```diff
@@ -81,13 +82,16 @@ Maintainers are free to use stock-extracted blobs, blobs from other devices or l
 * Official builds can be released as weekly, monthly or nightlies when needed but at least they must be released on every crDroid minor version update
 * Selinux can be permissive during initial bringup days, only in beta builds. It must be enforced in every official release
 * Sepolicy must meet security standards, it must cover every system and vendor component (no random denials), however it must not contain any rules for 3rd party apps
+* Neverallows MUST NOT be masked using *SELINUX_IGNORE_NEVERALLOWS*
+* Maintainers must not disable any permissions related checks, particularly alter *ro.control_privapp_permissions*
 * Official builds must not be released with gapps unless it is impossible to flash them without bootloop
-* If released with Gapps, you mush include our gapps version found at our [GitLab](https://gitlab.com/crdroidandroid/android_vendor_gapps)
+* If releasing with Gapps, you mush include our gapps version found at our [GitLab](https://gitlab.com/crdroidandroid/android_vendor_gapps)
```
Comment (DarkJoker360) [inline README.md]: typo at "mush"

Comment (xNombre) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ Following crDroid 8.0 release all rules are enforced.
 
 ### General rules
 * All maintainers must have knowledge about source control tools such as *git* and *repo*.
-* All maintainers must release device sources **publicly** at crDroid Android organization on Github, here includes common device tree if present, device tree, vendor and kernel
-* In the case of incompatibilities with current already hosted device sources, maintainers will need to create a dedicated branch for their devices on said repos
+* All maintainers must release device sources **publicly** at crDroid Android organization on Github, here includes common device tree (if present), device tree, vendor and kernel
+* All sources must be fully synced (pushed to GitHub) **prior to** every official build release
```
Comment (gwolf2u) [inline README.md:14]: How to validate?

File: README.md [hunk 3]
```diff
@@ -62,7 +63,7 @@ Commits must preserve proper and informative naming, e.g. *'sm6150-common: Add m
     * Must be operational and support same range of satellite types as stock
 * **Storage**
     * Every storage memory must be working, internal and external (like sd cards)
-    * Devices must support encryption. FDE devices should switch to FBE if possible, FBE encryption must be enabled by default
+    * Devices must support encryption. FDE devices should switch to FBE, FBE encryption must be enabled by default
```
Comment (gwolf2u) [inline README.md:68]: keep if possible for now as support is getting dropped with A13

File: README.md [hunk 4]
```diff
@@ -81,13 +82,16 @@ Maintainers are free to use stock-extracted blobs, blobs from other devices or l
 * Official builds can be released as weekly, monthly or nightlies when needed but at least they must be released on every crDroid minor version update
 * Selinux can be permissive during initial bringup days, only in beta builds. It must be enforced in every official release
 * Sepolicy must meet security standards, it must cover every system and vendor component (no random denials), however it must not contain any rules for 3rd party apps
+* Neverallows MUST NOT be masked using *SELINUX_IGNORE_NEVERALLOWS*
+* Maintainers must not disable any permissions related checks, particularly alter *ro.control_privapp_permissions*
 * Official builds must not be released with gapps unless it is impossible to flash them without bootloop
-* If released with Gapps, you mush include our gapps version found at our [GitLab](https://gitlab.com/crdroidandroid/android_vendor_gapps)
+* If releasing with Gapps, you must include our gapps variant found at our [GitLab](https://gitlab.com/crdroidandroid/android_vendor_gapps)
```
Comment (gwolf2u) [inline README.md]: suggest to drop this
we can't validate and does less work for us to maintain gapps

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (xNombre) [reply README.md]: As long as @neobuddy89 wants to maintain our gapps I would keep it.

Comment (gwolf2u) [reply README.md]: we done even have a 12.1 branch
as I said, less work, better for us
https://gitlab.com/crdroidandroid/android_vendor_gapps

File: README.md (hunk 2 above)
Comment (xNombre) [reply README.md:14]: We can't validate any of these points as maintainers are building on their own computers, we just have to trust them :)
Also, from time to time we should check a few devices chosen randomly from ota repository and "punish" maintainers if needed.

File: README.md (hunk 4 above)
Comment (xNombre) [reply README.md]: Im waiting for Pranav's answer, if he doesnt want to maintain it I will change this rule to enforce the most basic variant of gapps.

File: README.md (hunk 3 above)
Comment (xNombre) [reply README.md:68]: It is not needed as per 'should' is already a suggestion rather than an enforcement.

File: README.md (hunk 4 above)
Comment (xNombre) [reply README.md]: Removed our gapps reference.

**Review by gwolf2u — APPROVED** [review]

**Comment by gwolf2u** [comment]
OK from me
Squash merge

**Pull request merged**